.github/workflows/build-test-package.yml

```diff
@@ -7,6 +7,8 @@ jobs:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@d2f5d1894abf01a93cc6581c3b2f17b508170ce3
 
   python-build-workflow:
-    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@246883ef3828fc00219145aeec9efa67b9889d0b
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@d2f5d1894abf01a93cc6581c3b2f17b508170ce3
+    with:
+      itk-python-package-tag: v5.4rc02
```

Comment:
What is the right way to get 5.4RC2?

Reply:
Currently getting:
```log
CMake Error at CMakeLists.txt:31 (find_package):
-- Configuring incomplete, errors occurred!
  Could not find a configuration file for package "ITK" that is compatible
  with requested version "5.4".

  The following configuration files were considered but not accepted:

    /work/ITK-cp38-cp38-manylinux_2_28_x64/ITKConfig.cmake, version: 5.3.0



Traceback (most recent call last):
  File "/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/setuptools_wrap.py", line 666, in setup
    env = cmkr.configure(
  File "/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/cmaker.py", line 357, in configure
    raise SKBuildError(msg)
```

Reply:
Point to the commit, 3f63de3, which is the current tip of the `v5.4.0` branch: https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/tree/v5.4.0

Reply:
This seems to have worked. The error is now `ImportError: cannot import name '_POLICIES' from 'auditwheel.policy' (/opt/python/cp38-cp38/lib/python3.8/site-packages/auditwheel/policy/__init__.py)`.